DependencyInjection/DoctrineExtension.php

```diff
@@ -345,6 +345,7 @@ protected function ormLoad(array $config, ContainerBuilder $container)
         $loader->load('orm.xml');
 
         if (class_exists(AbstractType::class)) {
+            $loader->load('orm-form.xml');
             $container->getDefinition('form.type.entity')->addTag('kernel.reset', ['method' => 'reset']);
```

Comment:
The tag can be added to `orm-form.xml`. There is no need anymore to add it programmatically.